src/styles/theme.js

```diff
@@ -37,14 +41,22 @@ const breakpoints = {
   desktop: 1200,
 };
 
-export const getTheme = (userColors) => {
-  const colors = { ...universalColors, ...userColors };
+export const createTheme = (options = {}) => {
+  const { colors, ...rest } = options;
```

Comment:
Should maybe default `colors` here to an empty object in case it's not provided to avoid an error on the next line when we do an object spread on it.